example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java

```diff
@@ -33,6 +36,13 @@
   }
 
   public static void main(String[] args) throws Exception {
+    AddressUtils utils = new AddressUtils(Network.TESTNET);
+    String testPublicKey = ECKeyPair.publicKeyFromPrivate(TestPrivateKey);
+    if (!TestAddress.equals(utils.generateFromPublicKey(testPublicKey))) {
+      System.out.println(
+          "private key " + TestPrivateKey + " and address " + TestAddress + " are not match");
```

Comment:
Must not print private key

Reply:
https://github.com/nervosnetwork/ckb-sdk-java/pull/351/files/ba349e17a157811af5764b11886f420f7ba76300#diff-71ff690890b485a4a8e017d78306c475L23 just print example private key, Will there be any harm?

Reply:
got it, I will fix this.

Reply:
updated: [`b4b30c5` (#351)](https://github.com/nervosnetwork/ckb-sdk-java/pull/351/commits/b4b30c5f120a43e8ba561108710a1c5259c4aef5)

Reply:
I think `System.out.println("Private key and address (" + TestAddress + ") are not match");` is OK.

Reply:
The last four characters of the private key are added for the convenience of debugging when there are multiply addresses

Reply:
I think security is more important than debugging convenience and the probability of this error is very small.